### PR TITLE
Plt 6453: Migrate installed_incoming_webhooks.jsx to be pure and use Redux

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Migrate installed_incoming_webhooks.jsx and installed_incoming_webhook.jsx to be pure and use Redux 
#### Ticket Link
#6449
#6453 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Added or updated unit tests (required for all new features)
- [x] Has redux changes (please link)

### CODE REVIEW

File: routes/route_integrations.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default {
             path: 'incoming_webhooks',
             indexRoute: {
                 getComponents: (location, callback) => {
-                    System.import('components/integrations/components/installed_incoming_webhooks').then(RouteUtils.importComponentSuccess(callback));
+                    System.import('components/integrations/components/installed_incoming_webhooks/index.js').then(RouteUtils.importComponentSuccess(callback));
```
Comment (sudheerDev) [inline routes/route_integrations.jsx]: I get an error if I remove `/index.js` which is how I had in my PR before on mattermost-server repo. 

`ERROR in ./components/integrations/components/installed_incoming_webhooks.jsx
Module build failed: Error: ENOENT: no such file or directory, open '/Users/sudheer/go/src/github.com/mattermost/mattermost-webapp/components/integrations/components/installed_incoming_webhooks.jsx'`

Comment (sudheerDev) [reply routes/route_integrations.jsx]: After updating master and cleaning my repo it started to work fine without the file extension. So I reverted this change.

**Comment by hmhealey** [comment]
@sudheerDev Looks like you've got a style error
```
$ eslint --ext ".jsx" --ignore-pattern node_modules --quiet .

/home/jenkins/workspace/mattermost-webapp_PR-60-UHGHCIH5ONU3AD3H6YUFXGSLO42ZNQTL75BOK36JMNTXFC2ECL7Q/mattermost-webapp/components/integrations/components/installed_incoming_webhooks/installed_incoming_webhooks.jsx
  94:13  error  'displayNameB' is never reassigned. Use 'const' instead  prefer-const

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
```

**Comment by sudheerDev** [comment]
@hmhealey Yes fixed it. One more thing on master API for loading webhooks is `http://localhost:8065/api/v4/hooks/incoming?page=0&per_page=10000&team_id=eatxocwc3bg9ffo9xyybnj4omr` Whereas on my branch it is `http://localhost:8065/api/v4/hooks/incoming?page=0&per_page=60`. Not sure of the reason. Let me know if there is something I can do.

**Comment by sudheerDev** [comment]
@hmhealey Sorry one thing with the commit though. On master api for listing webhooks is pointed to 
`http://localhost:8065/api/v4/hooks/incoming?page=0&per_page=10000&team_id=eatxocwc3bg9ffo9xyybnj4omr` but whereas on my branch it is pointed to`http://localhost:8065/api/v4/hooks/incoming?page=0&per_page=60`.
I thought I will dig into it before this gets merged but couldn't get to it. Is there anything I was missing?

**Comment by hmhealey** [comment]
Oh nice catch. I definitely missed those. Do you mind submitting a followup PR to add those arguments to the function call? It already takes those same parameters, so they just need to be passed in

**Comment by sudheerDev** [comment]
@hmhealey Thanks fixed it.

**Pull request merged**